Assets/Shaders/Excellent/Excellent Icosahedron/Magic_moustache.shader

```diff
@@ -0,0 +1,67 @@
+﻿Shader "Ephemeral Laboratories/Magic moustache"
+{
+    Properties
+    {
+        _Colour ("_Colour", Color) = (1,1,1,1)
+        _MainTex ("Albedo (RGB)", 2D) = "white" {}
+        _Glossiness ("Smoothness", Range(0,1)) = 0.5
+        _Metallic ("Metallic", Range(0,1)) = 0.0
+        _Pivot ("Pivot", Vector) = (0.0,0.0,0.0,0.0)
+    }
+    SubShader
+    {
+        Tags
+        {
+            "RenderType" = "TransparentCutout"
+            "Queue" = "AlphaTest"
+        }
+        LOD 200
+        
+        CGPROGRAM
+        #pragma surface Surface Standard vertex:vert addshadow fullforwardshadows
+        #pragma target 4.0
+        #include "../../Common/ELMathUtilities.cginc"
+        sampler2D _MainTex;
+        float4 _Pivot;
+
+        struct Input
+        {
+            float2 uv_MainTex;
+        };
+        void vert(inout appdata_full v, out Input o) 
+        {
+            UNITY_INITIALIZE_OUTPUT(Input, o);
+            float rotation = _Time[1] * 25.0;
+            v.vertex.xyz -= _Pivot.xyz;
+            v.vertex.xyz = ELRotateAroundYInDegrees(v.vertex.xyz, -rotation);
+            v.vertex.xyz += _Pivot.xyz;
+
+            v.normal.xyz = ELRotateAroundYInDegrees(v.normal.xyz, -rotation);
+        }
+
+        half _Glossiness;
+        half _Metallic;
+        fixed4 _Colour;
+
+		bool IsInMirror()
```

Comment:
I recently added this one to the common set so you can now use `ELIsInMirror`.

Reply:
Also there's something going on with indenting in this file. It could be tabs again.